helm-charts/falcon-sensor/README.md

```diff
@@ -179,6 +180,72 @@ For a complete listing of configurable parameters, run the following command:
 helm show values crowdstrike/falcon-sensor
 ```
 
+#### Note about --set
+
+If you need to provide a list of values to a `--set` command, you need to escape the commas between the values e.g. `--set falcon.tags="tag1\,tag2\,tag3"`
+
+#### Note about container.image.sensorResources
+
+When setting `container.image.sensorResources`, the simplest method would be to provide a values file to the `helm install` command.
+
+Example:
+
+```bash
+helm upgrade --install falcon-helm crowdstrike/falcon-sensor \
+    --set node.enabled=false
+    --set container.enabled=true \
+    --set falcon.cid="<CrowdStrike_CID>" \
+    --set container.image.repository="<Your_Registry>/falcon-sensor"
+    --values local.yaml
```

Comment:
```suggestion
    --values values.yaml
```
This just lines up with the helm examples even though the filename doesn't matter.